src/components/PaymentMethod.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { TabPane } from "reactstrap";
+import { PaymentMethod as PaymentMethodDef } from "../generated/definitions/PaymentMethod";
+import { useTranslation } from "react-i18next";
+import { format, parseISO } from "date-fns";
+
+type PaymentMethodProps = {
+  el: PaymentMethodDef;
+  index: number;
+  key?: number;
+};
+
+function PaymentMethod(element: PaymentMethodProps): JSX.Element {
```

Comment:
All these functional components should have a type like:

```suggestion
const PaymentMethod: React.FunctionComponent<PaymentMethodProps> = (props) => {
```

please refer to https://github.com/piotrwitek/react-redux-typescript-guide#reactfcprops--reactfunctioncomponentprops for typiings react components

see for example:
https://github.com/pagopa/io-app/blob/master/ts/components/box/InfoBox.tsx#L28